docs/src/pages/components/dialogs/ConfirmationDialog.js

```diff
@@ -61,8 +61,6 @@ function ConfirmationDialogRaw(props) {
 
   return (
     <Dialog
-      disableBackdropClick
-      disableEscapeKeyDown
```

Comment:
As far as I could see this prop (`disableEscapeKeyDown`) is not deprecated. Why was it removed?

Reply:
It was removed on next too because we don't listen to the onClose event. The only impact is about the stop propagation behavior when <kbd>Escape</kbd> is pressed.